Header header
float32[1] gait_count
float32[1] gait_phase
float32[6] swing_phase
float32[6] stance_phase
float32[6] touch_possibility
